village/src/components/styles/S_NavBar.js

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components'
+
+const NavBar = styled.nav `
+  width: 100%;
+  height: 40px;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  font-size: 18px;
+  padding: 10px;
+  background: #224de3
+
+  a {
+    text-decoration: none;
+    color: white;
+  }
+
+  .active {
```

Comment:
Well done using the `.active` classname from the `NavLink`. This is why we choose `NavLink` over `Link` for navigation bar.